pyproject.toml

```diff
@@ -13,8 +13,17 @@ repository = "https://github.com/strawberry-graphql/strawberry"
 documentation = "https://strawberry.rocks/"
 
 classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
     "Topic :: Software Development :: Libraries",
-    "Topic :: Software Development :: Libraries :: Python Modules"
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
```

Comment:
@jkimbo do these change make sense to you?

Reply:
poetry adds them automatically: https://python-poetry.org/docs/pyproject/#classifiers

so maybe we don't the ones for python versions

Reply:
![image](https://user-images.githubusercontent.com/667029/130360765-63b3e805-a973-462b-a2ca-7ab97d9434f2.png)
yes, I'll remove the ones for python version :)